config/default/file.settings.yml

```diff
@@ -6,3 +6,10 @@ description:
 icon:
   directory: core/modules/file/icons
 make_unused_managed_files_temporary: true
+filename_sanitization:
```

Comment:
Do we want to adopt any of this now or later? I just got done explaining to a customer that though not required, the best practice is to have clean filenames so the encoded spaces issue isn't a big deal.

https://www.drupal.org/node/2972665

Reply:
it is off by default and is an opt-in feature

Reply:
At EoD the team wasn't interested in this now and didn't think a placeholder issue was worth it.